docker/docker-compose-tests.yml

```diff
@@ -16,15 +16,16 @@ services:
     container_name: es01
     environment:
       - discovery.type=single-node
+      - http.port=19200
```

Comment:
Just curious, was there a need to change the where Elasticsearch listens inside the container?

Reply:
I put a bit in the comment. We use a default bridge which other containers can share. i was thinking about it more on my walk this morning and i think we should just use our own bridge network for testing so this does not happen.

Reply:
Oh I missed that. I think we can also use our own bridge network e.g. as seen in the Elasticsearch Docker image [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-compose-file). Either way I am fine.

Reply:
yea I decided this was a better solution, just in case we add more nodes eventually, we dont want to have to continue to choose ports-o-fun. im running the it tests now on the change :)